ajax/libs/nerd-fonts/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "nerd-fonts",
+  "description": "Iconic font aggregator, collection, and patcher: 39 patched fonts (87k+ variations). Includes popular glyph collections such as Font Awesome & fonts such as Hack",
+  "keywords": [
+    "fonts",
+    "patched-fonts",
+    "powerline",
+    "shell",
+    "statusline",
+    "python",
+    "iconic-fonts",
+    "patcher",
+    "font-awesome",
+    "octicons"
+  ],
+  "author": "Ryan L McIntyre",
+  "license": "MIT",
+  "homepage": "https://github.com/ryanoasis/nerd-fonts#readme",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/ryanoasis/nerd-fonts.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/ryanoasis/nerd-fonts.git",
+    "fileMap": [
+      {
+        "basePath": "patched-fonts",
+        "files": [
+          "**/*"
```

Comment:
There are a lot of  *.md files, some *.ps1 files. Do we need them?

Reply:
The md files give useful context on each of the fonts (source, licenseing, etc.) so whilst they won't be available on the CDN they're useful to keep in the repo. I'm not sure what the ps1 files are, but I think its probably safer to just mirror this entire dir.

Reply:
For your discretion Matt, however, I personally never used for md file for any imports in apps. For information yes, but not in code

Reply:
Yeah, they won't even be available on the CDN, so you couldn't use them. I just think they should be copied across into our repo to give context on all these font files as it's a big library.